webapp/channels/src/components/admin_console/system_users/system_users_filters_popover/system_users_filter_team/index.tsx

```diff
@@ -45,9 +45,9 @@ export function SystemUsersFilterTeam(props: Props) {
 
     const [error, setError] = useState('');
 
-    const [list, setList] = useState<OptionsType<OptionType>>();
+    const [list, setList] = useState<Options<OptionType>>();
     const [pageNumber, setPageNumber] = useState(0);
-    const [value, setValue] = useState<ValueType<OptionType>>(getDefaultSelectedTeam(props.initialValue, props.initialLabel));
+    const [value, setValue] = useState<OnChangeValue<OptionType, false>>(getDefaultSelectedTeam(props.initialValue, props.initialLabel));
```

Comment:
I see in several places we set this `OnChangeValue` type to `true` or `false` instead of `boolean`. Is that intended?

Reply:
Weird way to use types, but I guess it is the way the library wants it to be :stuck_out_tongue: